src/tox/config/__init__.py

```diff
@@ -663,11 +663,14 @@ def passenv(testenv_config, value):
         value = list(itertools.chain.from_iterable([x.split(" ") for x in value]))
 
         passenv = {
-            "PATH",
-            "PIP_INDEX_URL",
+            "CURL_CA_BUNDLE",
             "LANG",
             "LANGUAGE",
             "LD_LIBRARY_PATH",
+            "PATH",
+            "PIP_INDEX_URL",
+            "REQUESTS_CA_BUNDLE",
```

Comment:
I don't think tox should know things about `requests`

Reply:
@asottile If I remember well, even pip is using requests more recently. If we do not include it, it will likely fail to run even pre-commit -- in fact that was why drove me to add them, I was not able to `tox -e lint` without them.

Reply:
pip vendors a ca-cert bundle, which should be active without needing anything right?

Reply:
Well not really behind a corporate proxy; as the ca-cert bundle obviously doesn't know about corporate proxy certificates. These usually are set as env-vars though.